### PR TITLE
(maint) Add stretch support

### PR DESCRIPTION
**Pull request by mwaggett**
This commit adds support for Debian 9, codename stretch.

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -75,7 +75,7 @@
             },
           }
         }
-        'wheezy', 'jessie', 'precise','quantal','raring','saucy', 'trusty', 'utopic': {
+        'wheezy', 'jessie', 'stretch', 'precise','quantal','raring','saucy', 'trusty', 'utopic': {
           $java =  {
             'jdk' => {
               'package'          => 'openjdk-7-jdk',
```
Comment (underscorgan) [inline manifests/params.pp]: stretch doesn't have java7 packages

Comment (mwaggett) [reply manifests/params.pp]: welp

Comment (mwaggett) [reply manifests/params.pp]:
```
root@lion:~# dpkg -l | grep jdk
ii  openjdk-6-jre:amd64                6b38-1.13.10-1~deb7u1                   amd64        OpenJDK Java runtime, using Hotspot JIT
ii  openjdk-6-jre-headless:amd64       6b38-1.13.10-1~deb7u1                   amd64        OpenJDK Java runtime, using Hotspot JIT (headless)
ii  openjdk-6-jre-lib                  6b38-1.13.10-1~deb7u1                   all          OpenJDK Java runtime (architecture independent libraries)
ii  openjdk-7-jdk:amd64                7u171-2.6.13-1~deb8u1                   amd64        OpenJDK Development Kit (JDK)
ii  openjdk-7-jre:amd64                7u171-2.6.13-1~deb8u1                   amd64        OpenJDK Java runtime, using Hotspot JIT
ii  openjdk-7-jre-headless:amd64       7u171-2.6.13-1~deb8u1                   amd64        OpenJDK Java runtime, using Hotspot JIT (headless)
```

Comment (underscorgan) [reply manifests/params.pp]: that's from a jessie repo, you can tell by the deb8u1. If you get a clean stretch vm there's no openjdk-7-jdk

Comment (underscorgan) [reply manifests/params.pp]: I don't know that all the vars are going to line up, but might be able to pull some info from https://github.com/puppetlabs/puppetlabs-java/blob/master/manifests/params.pp#L158-L173

Comment (mwaggett) [reply manifests/params.pp]: ahh, okay, I'll run with that! Thanks!

**Comment by mwaggett** [comment]
@underscorgan fixed!

**Review by underscorgan — APPROVED** [review]

**Comment by underscorgan** [comment]
@mwaggett this is targeting the wrong branch

**Comment by underscorgan** [comment]
You need to target https://github.com/underscorgan/puppetlabs-java/commits/maint/master/fix_version_darwin

**Comment by mwaggett** [comment]
oops, opening new PR now

**Pull request merged**